service/history/queues/metrics.go

```diff
@@ -38,7 +38,7 @@ const (
 
 	TaskFailures                    = "task_errors"
 	TaskDiscarded                   = "task_errors_discarded"
-	TaskSkipped                     = "task_skipped"
```

Comment:
Probably keep this one. It will be used after the metrics client interface is deprecated. Currently is a copy of `metrics.TaskSkipped`.

This metric is emitted when mutable state is stale compared to the task. Maybe we should rename it.